spring-kafka/src/main/java/org/springframework/kafka/support/DefaultKafkaHeaderMapper.java

```diff
@@ -31,6 +31,7 @@
 import org.apache.kafka.common.header.Headers;
 import org.apache.kafka.common.header.internals.RecordHeader;
 
+import org.springframework.kafka.support.serializer.SerializationUtils;
```

Comment:
We need to be sure that there is no package tangle . Usually sub-packages uses their parents.